source/playbooks/refining-and-improving.rst

```diff
@@ -23,7 +23,7 @@ Retrospectives help teams identify areas of improvement in workflows. Each time
 Configuring retrospectives before a run
 ----------------------------------------
 
-Open Playbooks and select the **Playbooks** tab. Select the playbook and then select **Edit**. Select **Templates** and scroll down to the **Retrospective Reminder Interval** and **Retrospective Template** fields.
+Open Playbooks and select the **Playbooks** tab. Select the playbook and then select **Edit**. Select **Retrospective**. Move the toggle to **Enable Retrospective**.
```

Comment:
It appears to take 6 "clicks" to configure a retro before a run. You've done a good job of documenting what a user specifically needs to do, but I wonder if there's an opportunity in the future to revisit and streamline the process overall through a UX redesign.

Reply:
Definitely worth streamlining it. I know there are ongoing efforts to refactor parts of playbooks. I'm happy to raise this with the team. :)